src/server/pegasus_server_impl_init.cpp

```diff
@@ -693,6 +745,15 @@ pegasus_server_impl::pegasus_server_impl(dsn::replication::replica *r)
     _data_cf_opts.max_bytes_for_level_base = FLAGS_rocksdb_max_bytes_for_level_base;
     _data_cf_opts.max_bytes_for_level_multiplier = FLAGS_rocksdb_max_bytes_for_level_multiplier;
 
+    // open db with key-value separation option (rocksdb blobdb)
+    _data_cf_opts.enable_blob_files = FLAGS_rocksdb_enable_blob_files;
```

Comment:
What happen if enable this switch on an already running rocksdb instance? Will the data to be lost?

Reply:
`enable_blob_files` will affects  flush/compaction task after your change, and will not  interrupting ongoing background tasks.   
If your turn it from 'true' to 'false', and at the same time  `enable_blob_garbage_collection=true`.:
The background Blob GC thread will continue to scan old files. When the garbage rate of the entire file exceeds the threshold (`blob_garbage_collection_force_threshold`), the surviving entries in the file will be moved to the new SST (not the new blob), and then the original file will be deleted.
On the other hand, if `enable_blob_garbage_collection=false`, or the keys of this blob file remain unchanged for a long time, the blob file  will stay in rocksdb for a long time.

Reply:
@ninsmiracle If `enable_blob_files` is set as true at the beginning, I wrote `k1` -> `v1` to a table. Then set it to false and reboot the replica servers, can I read the former `k1` from the DB? How it act if the data in both SST and blob files?